1-mandatory/2-people-i-know.js

```diff
@@ -379,13 +389,25 @@ First, I want you to find all of my friends who are 35 or older.
 
 let thirtyFiveOrOlder = [];
 
+thirtyFiveOrOlder = friends.filter((friend) => {
```

Comment:
Nitpick: For a single statement you don't need to use the full bracket notation of arrow functions. It's more readable to write
```
friends.filter(firend => friend.age >= 35);
```